miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/Dtos.java

```diff
@@ -766,4 +767,53 @@ public static LibraryAdditionalInfo to(LibraryAdditionalInfoDto from) {
     return to;
   }
 
+
+  public static PoolOrderDto asDto(PoolOrder from) {
+    PoolOrderDto dto = new PoolOrderDto();
+    dto.setId(from.getId());
+    dto.setParameters(asDto(from.getSequencingParameter()));
+    dto.setPartitions(from.getPartitions());
+    dto.setCreatedById(from.getCreatedBy().getUserId());
+    dto.setCreationDate(dateTimeFormatter.print(from.getCreationDate().getTime()));
+    dto.setUpdatedById(from.getUpdatedBy().getUserId());
+    dto.setLastUpdated(dateTimeFormatter.print(from.getLastUpdated().getTime()));
+    return dto;
+  }
+
+  public static Set<PoolOrderDto> asPoolOrderDtos(Collection<PoolOrder> from) {
+    Set<PoolOrderDto> dtoSet = Sets.newHashSet();
+    for (PoolOrder kd : from) {
+      dtoSet.add(asDto(kd));
+    }
```

Comment:
Using 'kd' is ambiguous as we normally use that for kitDescriptor, but not a dealbreaker.